src/shared/components/ContextMenu/ContextMenu.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react'
+import ContextMenu, { ContextMenuItem } from './ContextMenu'
+import { OverlayManagerProvider, useContextMenu } from '@/hooks'
+import { Button } from '@/shared/components'
+import { Meta, Story } from '@storybook/react'
+
+export default {
+  title: 'Shared/ContextMenu',
+  component: ContextMenu,
+  decorators: [
+    (Story) => (
+      <OverlayManagerProvider>
+        <Story />
+      </OverlayManagerProvider>
+    ),
+  ],
+} as Meta
+
+const Template: Story = (args) => {
+  const {
+    openContextMenu: leftOpenContextMenu,
+    closeContextMenu: leftCloseContextMenu,
+    contextMenuOpts: leftContextMenuOpts,
+  } = useContextMenu()
+  const {
+    openContextMenu: rightOpenContextMenu,
+    closeContextMenu: rightCloseContextMenu,
+    contextMenuOpts: rightContextMenuOpts,
+  } = useContextMenu()
+  return (
+    <>
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Button onClick={leftOpenContextMenu}>Open menu on the left side</Button>
+        <Button onClick={rightOpenContextMenu}>Open menu on the right side</Button>
+      </div>
+      <div>
+        <ContextMenu contextMenuOpts={leftContextMenuOpts} {...args}>
```

Comment:
What's the difference between this and the other menu? I know they're titled left and right but other than that I think they are the same?

Reply:
I guess I just don't fully understand

Reply:
The only purpose of two menus in stories is to represent how it behaves when there is more than one instance of it in view, e.g. view with video tiles, where every tile has its own context menu and we are jumping from one menu to another, then we can observe enter/exit transition between them.

I could remove it and leave only one menu in stories. What do you think?

Reply:
Ah ok, it just gave me the impression they should be somehow different but that makes sense